cms/models.py

```diff
@@ -319,6 +319,29 @@ def get_context(self, request, *args, **kwargs):
             "catalog_page": CatalogPage.objects.first(),
         }
 
+    @route(r"^certificates/program/([A-Za-z0-9-]+)/?$")
+    def program_certificate(self, request, uuid, *args, **kwargs):
```

Comment:
why are we keeping them under `HomePage` over there ? should not it be separated ?

Reply:
Moved under the new `CertificateIndexPage` model.